proto/pki/tx.proto

```diff
@@ -21,51 +21,51 @@ service Msg {
 
 message MsgProposeAddX509RootCert {
   string signer = 1 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.moretags) = "validate:\"required\""];
-  string cert = 2 [(gogoproto.moretags) = "validate:\"required\""];
+  string cert = 2 [(gogoproto.moretags) = "validate:\"required,max=3500000\""];
 }
 
 message MsgProposeAddX509RootCertResponse {
 }
 
 message MsgApproveAddX509RootCert {
   string signer = 1 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.moretags) = "validate:\"required\""];
-  string subject = 2 [(gogoproto.moretags) = "validate:\"required\""];
-  string subject_key_id = 3 [(gogoproto.moretags) = "validate:\"required\""];
+  string subject = 2 [(gogoproto.moretags) = "validate:\"required,max=50\""];
+  string subject_key_id = 3 [(gogoproto.moretags) = "validate:\"required,max=150\""];
```

Comment:
According to https://www.rfc-editor.org/rfc/rfc3280#section-4.2.1.2
subject key ID should be either 160 bit or 60 bit. So, 150 may be too small.
It looks like other ways of generation of subject_key_id are also possible.
So, let's assign `max=256`.